aten/src/ATen/core/ivalue_inl.h

```diff
@@ -411,6 +411,11 @@ struct C10_EXPORT ivalue::Future : c10::intrusive_ptr_target {
     return fut;
   }
 
+  // Since this file cannot import CUDA depedency, use void* to represent CUDA
+  // stream, and will cast it into the correct stream type whenever use it.
```

Comment:
Add a comment here mentioning the default implementation is a no-op since it doesn't deal with any CUDA streams.